docs/about/index.rst

```diff
@@ -8,6 +8,30 @@ About
 Mission
 -------
 
+Fairlearn is an open source toolkit that seeks to empower data scientists and
+developers to assess and improve fairness of their AI systems.
```

Comment:
It sounds like a key audience for some Fairlearn stakeholders might be teams making and selling an AI service.  In that case, those teams often won't have full control over who uses their services, and how they are deployed downstream.  My assumption has been that this is sort of an edge case, but it's come up a few different ways in recent discussions.

Understanding "who is the user and what are they trying to do" seems especially important for exploring new lines of UX work, and how UX work is intertwined with understanding the organizational process and individual processes for particular applications and use cases. 👍

Reply:
I'll make a minimal edit here "their AI systems" -> "AI systems". I'll see if I can find a place further down to comment on the control over the deploment context.